Mathlib/Topology/UniformSpace/UniformConvergenceTopology.lean

```diff
@@ -1124,35 +1124,43 @@ section UniformComposition
 variable {α β γ ι : Type*} [UniformSpace β] [UniformSpace γ] {p : Filter ι}
 
 /-- Composing on the left by a uniformly continuous function preserves uniform convergence -/
-theorem UniformContinuousOn.comp_tendstoUniformly (s : Set β) (F : ι → α → β) (f : α → β)
-    (hF : ∀ i x, F i x ∈ s) (hf : ∀ x, f x ∈ s)
-    {g : β → γ} (hg : UniformContinuousOn g s) (h : TendstoUniformly F f p) :
+theorem UniformContinuousOn.comp_tendstoUniformly {s : Set β} {F : ι → α → β} {f : α → β}
+    (hF : ∀ i x, F i x ∈ s) (hf : ∀ x, f x ∈ s) {g : β → γ} (hg : UniformContinuousOn g s)
+    (h : TendstoUniformly F f p) :
     TendstoUniformly (fun i x => g (F i x)) (fun x => g (f x)) p := by
   rw [uniformContinuousOn_iff_restrict] at hg
   lift F to ι → α → s using hF with F' hF'
   lift f to α → s using hf with f' hf'
   rw [tendstoUniformly_iff_tendsto] at h
-  have : Tendsto (fun q : ι × α ↦ (f' q.2, (F' q.1 q.2))) (p ×ˢ ⊤) (𝓤 s) :=
+  have : Tendsto (fun q ↦ (f' q.2, (F' q.1 q.2))) (p ×ˢ ⊤) (𝓤 s) :=
     h.of_tendsto_comp isUniformEmbedding_subtype_val.comap_uniformity.le
```

Comment:
There is a step that could be factored out here, something like `tendstoUniformly.of_comp`: if `r` is a `UniformEmbedding`, then a sequence of functions converges uniformly iff its composition with `r` does. If we had that, we could prove this lemma more tidily without having to use `tendstoUniformly_iff_tendsto`. But perhaps that is a matter for another PR.

Reply:
I think I'll leave this for another PR as this one is already dragging on.